ci/checkLinks.ts

```diff
@@ -0,0 +1,185 @@
+import { readdirSync, statSync, readFileSync } from "fs";
+import path from "path";
```

Comment:
```suggestion
import { readdirSync, statSync, readFileSync } from "node:fs";
import path from "node:path";
```

And while you are at it, you might as well.